fix-epub-links.sed

```diff
@@ -0,0 +1,76 @@
+# sed script to change internal links in markdown file,
+# to links pandoc can use when generating EPUB from markdown.
```

Comment:
Can you just give an example of an accepted pandoc link? What we have VS what we want (what this script does).